### PR TITLE
Feature/localstorage refresh page

### PR DESCRIPTION
**Pull request by mikevelko**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -9,53 +9,60 @@ import PrivateRoute from './components/PrivateRoute'
...
+  useEffect(() => {
+    const data = (window.localStorage.getItem("isUserAuthenticated"));
+    console.log(JSON.parse(data));
+    SetIsUserAuthenticated(JSON.parse(data));
+    console.log(isUserAuthenticated);
+  }, [])
+
+  useEffect(() => {
+    window.localStorage.setItem("isUserAuthenticated", JSON.stringify(isUserAuthenticated));
+  })
```
Comment (FlowerAlex) [inline client/src/App.js:27]: could be added only isUserAuthenticated like the second argument on useEffect, but it works, fine

File: client/src/components/Nav.jsx [hunk 2]
```diff
@@ -14,7 +14,7 @@ function Nav(props) {
     return (
         <div className="profile-nav">
             <div className="main-navbar-left">
-                <h2>Bookly client</h2>
+                <Link to="/home"><h2>Bookly client</h2></Link>
```
Comment (FlowerAlex) [inline client/src/components/Nav.jsx:17]: Better use Typography component for text fields and add Link like Button with setting params: component and to, existing in material-ui

**Review by FlowerAlex — APPROVED** [review]
Clean and readable code, good job

File: client/src/App.js (hunk 1 above)
Comment (mikevelko) [reply client/src/App.js:27]: Will be added when the client token is implemented

File: client/src/components/Nav.jsx (hunk 2 above)
Comment (mikevelko) [reply client/src/components/Nav.jsx:17]: @FlowerAlex ok thanks, this might make code more readable

**Pull request merged**